app/AttributeTypes/DropdownSingleAttribute.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\AttributeTypes;
+
+use App\Attribute;
+use App\ThConcept;
+use App\Exceptions\InvalidDataException;
+
+class DropdownSingleAttribute extends AttributeBase
+{
+    protected static string $type = "string-sc";
+    protected static bool $inTable = true;
+    protected static ?string $field = 'thesaurus_val';
+    protected static bool $hasSelection = true;
+
+    public static function getSelection(Attribute $a) {
+        return ThConcept::getChildren($a->thesaurus_root_url, $a->recursive);
+    }
+
+    public static function fromImport(string $data) : mixed {
+        $concept = ThConcept::getByString($data);
```

Comment:
If I understand correctly, the import must contain the concept_url that looks like: 
https://spacialist.escience[...]/my_first_label#4631865316531653

Reply:
It is also possible to provide an existing label. `getByString` checks for a matching URI first and if none is found, it checks if any existing label (in any language) matches the string